maestro-ai/src/main/java/maestro/ai/DemoApp.kt

```diff
@@ -0,0 +1,190 @@
+package maestro.ai
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.arguments.argument
+import com.github.ajalt.clikt.parameters.arguments.multiple
+import com.github.ajalt.clikt.parameters.options.default
+import com.github.ajalt.clikt.parameters.options.flag
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.types.float
+import com.github.ajalt.clikt.parameters.types.path
+import kotlinx.coroutines.async
+import kotlinx.coroutines.runBlocking
+import maestro.ai.antrophic.Claude
+import maestro.ai.openai.OpenAI
+import java.io.File
+import java.nio.file.Path
+
+
+fun main(args: Array<String>) = DemoApp().main(args)
+
+/**
+ * This is a small helper program to help evaluate LLM results against a directory of screenshots and prompts.
+ *
+ * ### Usage examples
+ *
+ * ```console
+ * maestro-ai-demo *.png
+ * ```
+ *
+ * ```console
+ * maestro-ai-demo foo_bad_1.png
+ * ```
+ *
+ * ### Input format
```

Comment:
What should be the expected output? would be helpful to document that as well

Reply:
Added expected output.

Also, I think we should add a link where the fixtures can be downloaded from. Otherwise the barrier entry to this DemoApp is very high, because of the need to take the screenshots manually.

Currently the fixtures dataset I use is from https://github.com/mobile-dev-inc/copilot/pull/188.

I will upload that dataset to GCP and paste the link here. Do you see any problem with it? I think the customer apps that are in https://github.com/mobile-dev-inc/copilot/pull/188 are okay with that?

Reply:
Lets not public the GCS link for now, you can make it work for any storage link. Lets write a runbook internally for this in case some one from us wants to evaluate. This repository should ideally have access to general data (from open source apps) instead of customer apps.

Reply:
Good points, though I don't agree.

I'd prefer to have this dataset public, internal runbooks tend to rot and no one uses them unless really needed. But if we made this testing dataset public, anybody could play around with LLM outptus and submit a PR that improves it.

If this requires removing apps from our customers, and using screenshots from more popular, well known apps instead (like Uber, Bolt), then I still think it's worth the effort. We should strive to make working on Maestro possible and easy to people outside of @mobile-dev-inc.

PS 1 The perfect situation would be that generating screenshots for each app with `takeScreenshot`. But I think it's too much work to automate this, and what we have now is enough.

Reply:
That said: I will take no action for now.